test/models/network-v2.spec.ts

```diff
@@ -483,5 +483,36 @@ describe(`NetworkV2`, () => {
       })
     })
 
+    describe(`changeNetworkParameter() event emitted`, () => {
+      beforeEach(async () => {
+        const receipt = await Network_v2.deploy(web3Connection, {
+          networkTokenAddress: networkToken.contractAddress!,
+          councilAmount: 100,
+          requiredValidators: 5,
+          requiredStakes: 10,
+        });
+  
+        network = new Network_v2(web3Connection, receipt.contractAddress);
+  
+        await network.loadContract();
+      });
+  
+      it(`changeNetworkParameter() emits NetworkParamChanged event`, async () => {
+        const expectedOldVal = await network.councilAmount();
+        const expectedNewVal = expectedOldVal + 1;
+        const param = await network.PARAM_COUNCIL_AMOUNT();
+  
+        const result = await network.changeNetworkParameter(param, expectedNewVal);
```

Comment:
you should call `changeCouncilAmount()` since `NetworkV2` does not expose `changeNetworkParameter` to the public, to use `changeNetworkParameter()` you'll have to call `sendTx` by hand